tests/support/tap.rs

```diff
@@ -185,6 +185,10 @@ where
 
             buf.freeze()
         });
-        Box::new(self.0.request_body_async(req))
+        Box::new(
+            self.0
+                .request_body_async(req)
+                .map_err(|err| err.to_string()),
```

Comment:
TIOLI: I think this could also be 
```suggestion
                .map_err(ClientError::to_string),
```

Reply:
The compiler is not quite happy about that: 

```

Reply:
ah whoops, nevermind then!